src/VatRates/ResolvesVatRates.php

```diff
@@ -0,0 +1,10 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\Vat\VatRates;
+
+use DateTime;
+
+interface ResolvesVatRates
+{
+    public function getDefaultVatRateForCountry(string $countryCode, ?DateTime $dateTime = null): ?float;
```

Comment:
I'm wondering why a date would be necessary for checking a vat rate

Reply:
This is probably not a use case for us, but the API supports it so I wanted to add it for the sake of completeness.

VAT rates change over time, so this might be useful if you want to verify something from a few years ago.